packages/extension/src/components/icon/camera.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from "react";
+import { IconProps } from "./types";
+
+export const CameraIcon: FunctionComponent<IconProps> = ({
```

Comment:
@renfengshi Could you look at another icon file and follow the convention for icon files?

Reply:
@HeesungB Sorry, I cannot get what you mean. Is the color which does not follow?

Reply:
@HeesungB What should I do to resolve this conversation?